### ISSUE TITLE
UI design overhaul needed.

### ISSUE THREAD

**Issue by Mesum-Hussain** [scoute-dich/browser#997]
The browser is great and the fact that it adopted Material You so fast make it even better.
But I think it is very counter intuitive to use any very confusing.
When I open the browser, I land on this GitHub page of the browser instead of a search engine page like Google or DuckDuckGo, and I am not sure how to get started with my searches. 
I would suggest taking inspiration from the Via browser because it's the best phone browser out there in my opinion..... but it's not open source (that's the only issue I have with it).

**Comment by woheller69** [comment]
just like in any other browser you can simply type your search into the address bar. 
And you can set your start page as you like in settings ...

**Comment by Mesum-Hussain** [comment]
> just like in any other browser you can simply type your search into the address bar. 

But why do I need to guess that I have to replace the URL of this GitHub Repo in the address bar for conducting a search, instead of simply typing my query in the Search Engine page, as an average user? 

> And you can set your start page as you like in settings ...

Why is a Search Engine page not set as the startpage by default? Why does an average user using this browser for the first time, need to go into the settings to make this browser normal?

Why is this GitHub Repo set as the startpage by default? Who really needs to visit this repo so often? 
I understand that you want this repository to get some attention recognition and attention, but I think you are making it unnecessarily complicated for the users to use this browser. 
I would rather try to make it as simple as possible..... I mean it's just a browser. I just need the Search Engine page to type my query immediately after I click the app icon..... Don't want to bother replacing any URL in the address bar or changing the startpage.

Basically defaults should be the most appropriate option for the majority of the users, so they can simply get by even without changing them.

**Comment by scoute-dich** [comment]
I will point the startsite to the wiki and add some hints there. Thanks for your thoughts.
